packages/bitwarden/data_stream/collection/_dev/test/system/test-default-config.yml

```
@@ -0,0 +1,11 @@
+input: httpjson
+service: bitwarden
+vars:
+  url: http://{{Hostname}}:{{Port}}
+  client_id: xxxx
+  client_secret: xxxx
+  token_url: http://{{Hostname}}:{{Port}}/connect/token
+data_stream:
+  vars:
+    preserve_original_event: true
+    preserve_duplicate_custom_fields: true
```

```suggestion
    preserve_duplicate_custom_fields: true
assert:
    hit_count : xxx
```

There is a new capability in `httpjson` input type configs to assert on the total number of documents ingested. This helps in identifying failures later when something in pipeline changes.
Reference [PR](https://github.com/elastic/integrations/pull/5326)